pig-runtime/src/org/partiql/pig/runtime/FoldingDomainVisitorBase.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
```

Comment:
minor: It is ok to **not** have this on each file as long as we have the appropriate LICENSE file in our root repository :) 

It is noise that I have learned to ignore but still could make our files a little leaner :)

Reply:
IntelliJ started adding it automatically to all the new files I create in this project.  I guess I can look at disablign the option.